client/src/services/LoginService.js

```diff
@@ -0,0 +1,50 @@
+export default class LoginService {
+  static login = (data) => {
+    const formData = new FormData();
+
+    formData.append('username', data.username);
+    formData.append('password', data.password);
+
+
+    // Configure POST request options
+    const fetchOptions = {
+      method: 'POST',
+      body: formData,
+    };
+
+    return new Promise(async (resolve, reject) => {
+      try {
+        // Sends the POST Request
+        // Catch network errors and thow them to the catch block
```

Comment:
This is fine But this comment isnt needed here since the code its explain is very straight forward.

We can keep it though, so no need to resubmit code review.

Reply:
Most code is self documenting.

We'll typically explain business logic. Like the "why", no necessarily the "what".

Reply:
I’ll try to keep to more relevant comments moving forward. Thank you for the feedback.

Reply:
using "why" and "what" is a poor example.........more like a summary of what's going on. Some lines of code don't need to be summarized as their read almost like English.